generatebundlefile/go.mod

```
@@ -1,6 +1,8 @@
 module github.com/aws/eks-anywhere-packages/generatebundlefile
 
-go 1.21
+go 1.23
+
+toolchain go1.23.5
```

```suggestion
toolchain go1.23.7
```

Let's use the latest version?